daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/LanguageVersion.scala

```diff
@@ -69,6 +69,10 @@ object LanguageVersion {
   val EarlyAccessVersions: VersionRange[LanguageVersion] =
     StableVersions
 
+  // Only the current stable version, no older versions.
+  val StableOnlyVersions: VersionRange[LanguageVersion] =
+    StableVersions.copy(min = StableVersions.max)
+
```

Comment:
What is the use case of this ?

Reply:
I found that pretty weird this is potentially changing at each realize. 
Those constant are part of the Engine API, if there is not a solid reason to have a new one, the client should simply define is own range.

Reply:
To be able to run things without enabling loading of old LF packages. As a preview for removing support for old versions.

Reply:
> As a preview for removing support for old versions.
We do not really need to preview removing support for old versions. In general we will never do it. Th only use case I see in the foreseeable future (before daml 2) is canton that will not support version older that 14. 

In any case I do not really see the point to define a constant, the ledger can defined is own range.

Reply:
>  The only use case I see in the foreseeable future (before daml 2) is canton.

That's why I'm adding this now, and we can remove it once it's not useful. What's the reason to not define the range here?

Reply:
Canton will needs two ranges: 

```
  val CantonStableVersions: VersionRange[LanguageVersion] =
    StableRange.copy(min = v1_14)  
 
 val CantonDevVersions: VersionRange[LanguageVersion] =
    DevRange.copy(min = v1_14)
```

The min version should be fixed  and not changing every time we change `StableVersions.max` (which potentially changes at each release).

This is canton specific it should be defined in canton code, there is no reason to defined it here.